packages/smooth_app/lib/themes/theme_manager.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class UserThemePreference {
+  // ignore: non_constant_identifier_names
+  static String THEME_STATUS = 'THEMESTATUS';
+
+  Future<void> setDarkTheme(bool value) async {
```

Comment:
As said previously, should be part of `UserPreferences`.

Reply:
I'm not sure if we should write this in the "UserPreferences", since all logic there except of the MLKitState is only for data retrieval from the servers. It is also mentioned in the file at the top in a comment, that this file will be transferred.

"[...] this needs to be transferred to the openfoodfacts-dart plugin when ready".

Reply:
The fact is that we should not create a new class each time we need to put something in the `SharedPreferences`.
Especially if we want to avoid confusion with the preference tags, and as `UserPreferences` is initialized in `main.dart` we don't have to reopen asynchronously a `SharedPreferences`.

I assume the comment you're referring to is not 100% relevant anymore, except that this `user_preferences.dart` file has nothing to do in a `temp` folder.

I know that @PrimaelQuemerais is on his way to write a short recommendations / best practice manual, and I think the use of `SharedPreferences` should be part of it.

Reply:
Okay, I will wait for the guidelindes and revise it then